build_tools/benchmarks/suites/definitions/iree.py

```diff
@@ -3,8 +3,9 @@
 # Licensed under the Apache License v2.0 with LLVM Exceptions.
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Classes for IREE benchmark definitions."""
 
-from dataclasses import dataclass
+from dataclasses import dataclass, field
```

Comment:
I didn't mention it when you were importing dataclass directly because that is pretty obvious from the name, but the Google Python style guide says to only [import the package or module](https://google.github.io/styleguide/pyguide.html#22-imports) (with the exception of `Typing`). We generally exercise a fair amount of discretion in our adherence to the style guides (especially the Python one, which is quite silly in places), but I think it's right on this one. The unqualified name "field" is pretty ambiguous. I'd recommend importing the whole package here and using qualified names.

Reply:
Thanks for pointing it out! It's opposite the Java guide line.